### PR TITLE
Update SkipAwareHookableFeatureTester.php

### PR DESCRIPTION
**Pull request by Kibsoon**
Please keep the author. Remember about copywright!

### CODE REVIEW

File: src/Tester/SkipAwareHookableFeatureTester.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 /*
  * This file is part of the SkipExtension package.
  *
+ * (c) Kamil Kokot <kamil@kokot.me>
```
Comment (Zales0123) [inline src/Tester/SkipAwareHookableFeatureTester.php:6]: Missing blank line below.

Comment (pamil) [inline src/Tester/SkipAwareHookableFeatureTester.php:6]: Missing blank comment line below, just like:

```
 *
```

Comment (lchrusciel) [inline src/Tester/SkipAwareHookableFeatureTester.php:6]: CS!

**Review by lchrusciel — CHANGES_REQUESTED** [review]

File: src/Tester/SkipAwareHookableFeatureTester.php (hunk 1 above)
Comment (Zales0123) [reply src/Tester/SkipAwareHookableFeatureTester.php:6]: You can copy this one and past it below.

Comment (Kibsoon) [reply src/Tester/SkipAwareHookableFeatureTester.php:6]: Thank you bery much bro! I will fix that asap.

**Comment by Zales0123** [comment]
Btw, your branch name is not really meaningful :/ Be aware of that in the future.

**Review by pamil — CHANGES_REQUESTED** [review]
Can you update `SkipExtension.php` with the same change as well?

**Review by Zales0123 — CHANGES_REQUESTED** [review]
Btw, your branch name is not really meaningful :/ Be aware of that in the future.

**Review by lchrusciel — APPROVED** [review]

**Review by Zales0123 — APPROVED** [review]

**Pull request merged**